brands/amenity/bank.json

```diff
@@ -2254,6 +2254,15 @@
       "name": "Banorte"
     }
   },
+  "amenity/bank|Banpais": {
```

Comment:
For this one, it would be great to have a wikidata link.  Also please remove the redundant `matchNames` property.